tidb/yaml/scale-out.yaml

```diff
@@ -0,0 +1,21 @@
+tikv_servers:
```

Comment:
This yaml file name is too specified to scale-out, it could use to other things, like just testing multiple nodes in one server